### DISCUSSION TITLE
OpenSSL 3.0.7 performance Issue

### DISCUSSION THREAD

**Discussion by sghatuwa** [openssl/openssl#25481] [Q&A]
From the issues provided in repo, we know there present huge performance degradation in 3.0.x.

Initially, we created the certificate from 3.0.7, and faced SSL issues on verifying SSL certificate from 3.0.7. And after using OpenSSL 3.3.x for certificate verification, we didn't faced such performance issues.

So, is the performance issue fixed in latest 3.0.x version (**i.e. 3.0.15**)?

**Comment by mattcaswell** [comment]
For best performance you should use the latest stable release (3.3.2). There are many performance improvements in the 3.3 branch that have not been backported to the 3.0 branch.

Performance in 3.0.15 will be similar to 3.0.7.

**Reply by sghatuwa** [reply]
Thank you for your feedback. As 3.0 is LTS, but there present lot of performance issue in that version. So, Is there any plan for new LTS version release?
